explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -36,6 +36,9 @@ class CorrelationConfig(MetricConfig):
     use_z_score: bool = True
     no_human: bool = True
 
+    def to_metric(self):
+        pass
```

Comment:
```suggestion
        raise NotImplementedError
```

Reply:
And add type information to the function.